### DISCUSSION TITLE
Use tactital rmm in local network / VPN

### DISCUSSION THREAD

**Discussion by mathsyx69** [amidaware/tacticalrmm#404] [Q&A]
Hello, thank for your work, it's a good solution which have many fonctionnalities we search in you IT département.The only problem for us it's that solution work with public ip, for evident security reasons we can't use it in your company.
My question is: Is it possible to use tactical rmm in local network / through a vpn with local dns access?

Thank by advance

**Comment by dinger1986** [comment]
Yes it is.

You don’t need to open any ports just use an external domain that can be verified.

**Comment by wh1te909** [comment]
https://wh1te909.github.io/tacticalrmm/faq/#can-i-run-tactical-rmm-locally-behind-nat-without-exposing-anything-to-the-internet

**Reply by mathsyx69** [reply]
Thank 👍
